deployment/kustomizations/kustomization.yaml

```diff
@@ -0,0 +1,60 @@
+##### Dependancies used in this kustomization:
+# nginx.ingress
+# external-dns
+# cert-manager
+# kubectl kustomize
+
+#test output like this:
+#kubectl kustomize deployment/kustomizations --enable-helm > deployment/kustomizations/wrenai.kustimized.yaml
+#kubectl create namespace wren
+#kubectl apply -f deployment/kustomizations/wrenai.kustimized.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Namespace for all resources here and in the Helm charts
+namespace: wren
+helmCharts:
+
+  - name: postgresql
+    repo: https://charts.bitnami.com/bitnami
+    version: 14.3.3
+    releaseName: wren-postgresql
+    valuesFile: helm-values_postgresql_14.yaml
+    includeCRDs: true
+    # the Same Namespace
+    namespace: wren
+
+  - name: qdrant
+    repo: https://qdrant.github.io/qdrant-helm
+    version: 0.8.5
```

Comment:
Let's use `0.9.0` instead ? https://github.com/qdrant/qdrant-helm/releases/tag/qdrant-0.9.0

Reply:
Sure, I do not see a big problem. I haven't tested it with 0.9.0 though.